relayer/cmd/sync_beefy_commitment.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/sirupsen/logrus"
+	"github.com/snowfork/snowbridge/relayer/chain/ethereum"
+	"github.com/snowfork/snowbridge/relayer/relays/beefy"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func syncBeefyCommitmentCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sync-latest-beefy-commitment",
+		Short: "Sync beefy commitment on demand",
+		Args:  cobra.ExactArgs(0),
+		RunE:  SyncBeefyCommitmentFn,
+	}
+
+	cmd.Flags().String("config", "/tmp/snowbridge/beefy-relay.json", "Path to configuration file")
+	cmd.Flags().String("private-key", "", "Ethereum private key")
+	cmd.Flags().String("privateKeyFile", "", "The file from which to read the private key")
+	cmd.Flags().Uint64P("relay-block", "b", 0, "Relay block number which contains a Parachain message")
```

Comment:
I prefer this naming, since in the context of beefy, its always the relay chain 

```suggestion
	cmd.Flags().Uint64P("block-number", "b", 0, "Relay block number which contains a Parachain message")
```

Reply:
https://github.com/Snowfork/snowbridge/pull/1217/commits/98e3e74a58119717134797cfecec8a8895e43124